README.md

```diff
@@ -2,12 +2,37 @@
 
 This is a small simple service to mirror hex.pm to the local disk.
 
-It was written with [elixir.camp](http://elxiir.camp), where there is no internet connection, and so needs to be able to work completely offline.
+This was written for [elixir.camp](http://elxiir.camp),
+where there is no internet connection, and so needs to be
+able to work completely offline.
 
 ## Instructions
 
 1. Download it, setup deps and run
-2. Wait. By default it will start mirroring automatically and save to `./tarballs` As of July 2016, this will fetch ~700MB of data
+
+```bash
+git clone git@github.com:aussiegeek/hex_mirror.git
+npm install
+mix deps.get
+npm install
+mix phoenix.server
+```
+
+2. Wait. By default it will start mirroring automatically and save to
+`./tarballs`. As of July 2016, this will fetch ~700MB of data.
+
+3. People that want to use your mirror will need to do this
+(assuming your local address is 'thingy.local'):
+
+```bash
+mix hex.config mirror_url http://thing.local
```

Comment:
Now that I my copy finished downloading, I tried running this with:

```bash
mix hex.config mirror_url http://localhost:4000
```

and disbling my network. It didn't work. I got this error:

```bash
mix deps.get
** (Mix) The repository at http://localhost:4000 did not provide a signature for the registry because it returned http status code 404. This could be because of a man-in-the-middle attack or simply because the repository does not sign its registry. The signature verification check can be disabled by setting `HEX_UNSAFE_REGISTRY=1`.
```

When I tried setting `HEX_UNSAFE_REGISTRY=1` it didn't make any difference.

How should I be testing this?